keyboards/mechwild/sugarglider/config.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Kyle McCreery
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+#ifdef VIA_ENABLE
+#define DYNAMIC_KEYMAP_LAYER_COUNT 5
+#endif
+
+/* Matrix COL and ROW definitions */
+#define MATRIX_ROWS 9
+#define MATRIX_COLS 6
```

Comment:
```suggestion
```

Reply:
These aren't needed, and will be computed automatically, based on the info.json config.

Reply:
The pins arent defined in info.json as it is a custom matrix. Without these being defined it fails to compile.